mock-core-configs/etc/mock/eol/templates/mageia-7.tpl

```diff
@@ -1,9 +1,10 @@
 config_opts['chroot_setup_cmd'] = 'install basesystem-minimal rpm-build rpm-mageia-setup rpm-mageia-setup-build'
 config_opts['dist'] = 'mga7'  # only useful for --resultdir variable subst
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
-config_opts['releasever'] = '7'
+config_opts['releasever'] = '7.1'
```

Comment:
This should not be necessary.

Reply:
Reverted with f85dcd56f66c60824168def7998912f1874a1113